core/src/main/java/tc/oc/pgm/blockdrops/BlockDropsMatchModule.java

```diff
@@ -85,15 +85,18 @@ private void dropItems(BlockDrops drops, Location location, double yield) {
     Random random = match.getRandom();
     for (Map.Entry<ItemStack, Double> entry : drops.items.entrySet()) {
       if (random.nextFloat() < yield * entry.getValue()) {
-        location.getWorld().dropItemNaturally(location, entry.getKey());
+        location.getWorld().dropItemNaturally(location.toCenterLocation(), entry.getKey());
```

Comment:
I feel like the `toCenterLocation` should be done before calling `dropItems`, same with `dropExperience`